### ISSUE TITLE
displaying bug of github id

### ISSUE THREAD

**Issue by mackristof** [chamerling/QuickHubApp#14]
all works fine but : http://t.co/W7plubrD

**Comment by chamerling** [comment]
Easy to fix, thanks

**Comment by mobilemind** [comment]
Similar  issue, my github id is "mobilemind", but Issues menu items show up as "[obilemind/repoName] issueTitle" using QuickHub 1.4.

**Comment by chamerling** [comment]
@mobilemind Oh really, this is strange. In the issues items, this type of display pattern is for organizations i.e. [orgName/repoName] Title.
Can you attach a screenshot?

**Comment by mobilemind** [comment]
@chamerling Working fine now. Consider this closed.

I am unable to reproduce the Issues menu items issue. I tried going online/offline, sleep/wake, accessing repos with Gitbox app (Mac), SourceTree app (Mac), running bash scripts that called git, etc. THose were all things I was doing yesterday when I saw it consistently. All I can come up with now is maybe some sort of special character encoding problem from a weird character in an issue I've recently closed.
